public/app/features/alerting/unified/RuleList.test.tsx

```diff
@@ -287,7 +272,7 @@ describe('RuleList', () => {
     });
 
     mocks.api.fetchRules.mockImplementation((dataSourceName: string) => {
-      if (dataSourceName === GRAFANA_RULES_SOURCE_NAME) {
+      if (dataSourceName !== dataSources.prom.name) {
```

Comment:
Previous tests were setting up one specific datasource, but now it's using the same set of datasources for all tests. Rather than change the assertions within the tests, I've made the mocks only bother to respond properly when the same specific datasource (`datasources.prom`) is requested